discovery-provider/src/queries/prometheus_metrics_exporter.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from flask import Blueprint, Response
+from prometheus_client import (
+    CONTENT_TYPE_LATEST,
+    CollectorRegistry,
+    generate_latest,
+    multiprocess,
+)
+
+logger = logging.getLogger(__name__)
+
+bp = Blueprint("metrics", __name__)
```

Comment:
is it possible to change this name too to match the file? 🙂

Reply:
Renamed! 3d621206